packages/form-layout/src/vaadin-form-layout.js

```diff
@@ -254,13 +252,12 @@ class FormLayout extends ElementMixin(ThemableMixin(mixinBehaviors([IronResizabl
     // `super.ready()`, because `super.ready()` invokes property observers,
     // and the observer for `responsiveSteps` does CSS value validation.
     this._styleElement = document.createElement('style');
-    this.root.appendChild(this._styleElement);
+    this.appendChild(this._styleElement);
```

Comment:
Is this change intentional? Why don't we change to `this.shadowRoot` instead of legacy `this.root` field?
There could be logic in Flow counterpart to remove all children that will remove style node, too.

Reply:
The shadow root doesn't exist at this point yet (before `super.ready()`). With the current implementation `this.root` actually refers to `this`.

Reply:
Thanks, I totally missed that.